pydantic/main.py

```diff
@@ -853,6 +860,8 @@ def validate_model(  # noqa: C901 (ignore complexity)
                     errors.append(ErrorWrapper(ExtraError(), loc=f))
 
     for validator in model.__post_root_validators__:
+        if len(errors) and getattr(validator, SKIP_ON_FAILURE_KEY, False):
```

Comment:
```suggestion
        if errors and getattr(validator, SKIP_ON_FAILURE_KEY, False):
```

Reply:
Thanks! I shamefully didn't know that idiom.

Reply:
No worries! It's a tiny bit faster, and a little cleaner (in my opinion).